### ISSUE TITLE
Block barriers

### ISSUE THREAD

**Issue by LegendIsAwesomes** [TotalFreedom/TotalFreedomMod#678]
Block the items barriers from being placed. Since they're invisible they cannot be used to build and just are used to troll and grief.

**Comment by Wild1145** [comment]
Probably a good idea.

**Comment by OxLemonxO** [comment]
They are used for making parkour.
If you block them, block it in WE.

**Comment by WarrioR151** [comment]
Des please BLOCK it and in worldedit Too and please BLOCK in WE láva,water,endportal,etc..
Please BLOCK in se all BLOCK can lag server

**Comment by Wild1145** [comment]
Invisible blocks shouldnt really be used anywhere. They should be blocked here and in the WorldEdit config.

**Comment by tylerhyperHD** [comment]
Yeah, with great freedom comes great responsibility, and this isn't freedom, this is just annoying.

**Comment by JeromSar** [comment]
I think we should create a system that blocks specific blocks from being placed. Including barriers. :P
